backend/endpoints/volunteer.py

```diff
@@ -194,6 +198,33 @@ def get_volunteers_by_filters(filters, pages=0, per_page=10000):
 def deleteVolunteer(requestjson, volunteer_id):
         updateVolunteer(requestjson, volunteer_id, delete=True)
 
+def boolconv(source):
+    if type(source) is bool:
+        if source:
+            return 1
+        else:
+            return 0
+    else:
+        return source
+
+def volunteer_build_csv():
+    includelist = ['first_name', 'last_name', 'phone,telegram_id', 'address,zone_address',
```

Comment:
Why `'phone,telegram_id'` is one string? maybe they should be separated:
`'phone','telegram_id'`